### ISSUE TITLE
[docs] Troubleshooting for failed tablets leading to TS crash loops

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#4662]
After #3539 we will automatically trigger a remote bootstrap for most types of tablet data corruption or failures. However, there are still a number of cases where this automatic mechanism might not be able to help.

One major such case is anything that leads to a codepath that we've encoded as a crash (eg: CHECK or FATAL) or is an unknown unknown that causes a crash (eg: code bugs leading to SIGSEGV). Such crashes are likely repeatable, leading to the server being stuck in a crash loop, hence runtime automatic fixes would not be able to kick in, as the TS would not get to a steady state yet.

Steps to address this:
Normally, to fix this, we need to know the tablet(s) that are hitting these problems. What a user would need to do to address these is the following.

Assumptions
- the tablet UUID responsible for these failures is `FOO`
- the `fs_data_dirs` flag for these is `/mnt/disk1`

Steps
- Stop the tserver (to prevent new restarts, during operations)
- Find and remove all the tablet files: `find /mnt/disk1 -name '*FOO*' | xargs rm -rf`
- Start back the tserver

cc @rao-vasireddy

**Comment by ddorian** [comment]
@bmatican how will the user know which tablet is failing ? I'm guessing it will be in the logs ?

**Comment by bmatican** [comment]
@ddorian Yeah, ideally this would be something from the logs, in case of crash loops.

For particular types of inconsistencies, like rocksdb data/metadata, we have some tools that can be used to check those, even when the TS is offline, as they go over files. We should ideally expand on those to check all metadata required for bootstrap, but that's probably a separate task.
